src/sentry/integrations/repository/metric_alert.py

```diff
@@ -5,15 +5,21 @@
 
 from sentry.incidents.models.alert_rule import AlertRuleTriggerAction
 from sentry.incidents.models.incident import Incident
-from sentry.integrations.repository.base import BaseNewNotificationMessage, BaseNotificationMessage
+from sentry.integrations.repository.base import (
+    BaseNewNotificationMessage,
+    BaseNotificationMessage,
+    NotificationMessageValidationError,
+)
 from sentry.models.notificationmessage import NotificationMessage
 
 _default_logger: Logger = getLogger(__name__)
 
 
 @dataclass(frozen=True)
 class MetricAlertNotificationMessage(BaseNotificationMessage):
+    # TODO(Yash): do we really need this entire model, or can we whittle it down to what we need?
     incident: Incident | None = None
+    # TODO(Yash): do we really need this entire model, or can we whittle it down to what we need?
```

Comment:
nit: duplicate comment?

Reply:
No, both of these objects I'm hoping we can lean down to what we need!